src/catalogs/aviris/aviris_df/__init__.py

```diff
@@ -0,0 +1,144 @@
+from ftplib import FTP
+from functools import partial
+from pathlib import Path
+
+from geopandas import GeoDataFrame
+import pandas as pd
+
+from .converters import aviris_series_to_item
+
+
+AVIRIS_DESCRIPTION = "AVIRIS is an acronym for the Airborne Visible InfraRed Imaging Spectrometer. AVIRIS is a premier instrument in the realm of Earth Remote Sensing. It is a unique optical sensor that delivers calibrated images of the upwelling spectral radiance in 224 contiguous spectral channels (also called bands) with wavelengths from 400 to 2500 nanometers (nm). AVIRIS has been flown on four aircraft platforms: NASA's ER-2 jet, Twin Otter International's turboprop, Scaled Composites' Proteus, and NASA's WB-57. The ER-2 flies at approximately 20 km above sea level, at about 730 km/hr. The Twin Otter aircraft flies at 4km above ground level at 130km/hr. AVIRIS has flown all across the US, plus Canada and Europe. This catalog contains all AVIRIS missions from 2006 - 2019."
```

Comment:
Why not to use the [namespace package](https://docs.python.org/3/glossary.html#term-namespace-package) here? (just a curiosity question)